src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -99,7 +99,7 @@ private function populateEntrypointProperties(ApiResource $resourceMetadata, str
                 '@id' => \sprintf('#Entrypoint/%s', lcfirst($shortName)),
                 '@type' => $hydraPrefix.'Link',
                 'domain' => '#Entrypoint',
-                'rdfs:label' => "The collection of $shortName resources",
+                'rdfs:label' => "get{$shortName}Collection",
```

Comment:
Is this new value useful? Maybe can we remove it entirely.

Reply:
not sure indeed I'll check